src/main/java/domain/RandomNumberGeneratorImpl.java

```diff
@@ -0,0 +1,13 @@
+package domain;
+
+import java.util.Random;
+
+public class RandomNumberGeneratorImpl implements RandomNumberGenerator {
```

Comment:
[의견] 인터페이스를 분리해주셨네요 👍 클래스의 네이밍이나 테스트에서의 용도를 봤을 때 `RandomNumberGenerator`보다는 조금 더 숫자 범위를 만들어준다는 광의적인 네이밍을 사용해주면 좋을 것 같아요. 이런 건 어떨까요?

```suggestion
public class RandomNumberRangeGenerator implements NumberRangeGenerator {
```

> 완전 주관적인 네이밍이니, 로키 하고 싶은대로 변경하시면 될 것 같아요! 다만 이렇게 하면 `~Impl`을 사용하지 않아도 된다는 장점(?) 정도 있을 것 같네요.

Reply:
당시 페어와 네이밍을 고민했었는데 좋은 네이밍이군요!
Range 라는 단어가 정말 잘 어울리는것 같아요